runtime/compiler/env/VMJ9.cpp

```diff
@@ -4755,49 +4755,44 @@ TR_J9VMBase::targetMethodFromMethodHandle(TR::Compilation* comp, TR::KnownObject
    return NULL;
    }
 
-J9JNIMethodID*
-TR_J9VMBase::jniMethodIdFromMemberName(uintptr_t memberName)
+bool
+TR_J9VMBase::getMemberNameMethodInfo(
+   TR::Compilation* comp,
+   TR::KnownObjectTable::Index objIndex,
+   MemberNameMethodInfo *out)
    {
-   TR_ASSERT(haveAccess(), "jniMethodIdFromMemberName requires VM access");
-   return (J9JNIMethodID *)J9OBJECT_U64_LOAD(vmThread(), memberName, vmThread()->javaVM->vmindexOffset);
-   }
+   *out = {};
```

Comment:
This doesn't compile on z/OS:
```
13:10:07  "/jenkins/workspace/Build_JDK17_s390x_zos_Personal/openj9/runtime/compiler/env/VMJ9.cpp", line 4764.11: CCN5063 (S) The text "{" is unexpected.
13:10:07  CCN0793(I) Compilation failed for file /jenkins/workspace/Build_JDK17_s390x_zos_Personal/openj9/runtime/compiler/env/VMJ9.cpp.  Object file not created.
13:10:07  runtime/compiler/CMakeFiles/j9jit.dir/build.make:960: recipe for target 'runtime/compiler/CMakeFiles/j9jit.dir/env/VMJ9.cpp.o' failed
13:10:07  make[6]: *** [runtime/compiler/CMakeFiles/j9jit.dir/env/VMJ9.cpp.o] Error 12
```
@jdmpapin Is there another way to say this that will be accepted by older compilers?

Reply:
Should it be `{0}` perhaps?

Reply:
> The text "{" is unexpected.

suggests that we might get the same error from `{0}`. I think `{}` should work as an initializer though. I'll try this:
```
MemberNameMethodInfo zero = {};
*out = zero;
```
and of course check to make sure that it compiles on z/OS

Reply:
It looks like it should work to use
```
MemberNameMethodInfo zero = {0};
```
With `{}`, the build compiler complained that `zero` was potentially uninitialized

My personal z/OS build failed for a reason I don't understand. I *think* it's unrelated, but I've asked for some help confirming that it builds. I'll open a PR as soon as I get confirmation

Reply:
https://github.com/eclipse-openj9/openj9/pull/18087